libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/WPGutenbergWebViewActivity.java

```diff
@@ -140,13 +140,13 @@ private String getAuthenticationPostData(String authenticationUrl,
 
     @Override
     protected List<String> getOnGutenbergReadyExternalSources() {
-        String file = getFileContentFromAssets("unsupported-block-editor/remove-nux.js");
+        String file = getFileContentFromAssets("remove-nux.js");
         return Arrays.asList(file);
     }
 
     @Override protected List<String> getOnPageLoadExternalSources() {
         long userId = getIntent().getExtras().getLong(ARG_USER_ID, 0);
-        String file = getFileContentFromAssets("unsupported-block-editor/extra-localstorage-entries.js")
+        String file = getFileContentFromAssets("extra-localstorage-entries.js")
```

Comment:
These files live in the `resources/unsupported-block-editor` folder in `gutenberg-mobile`, but the AAR packaging (at least in the way it's configured [here](https://github.com/WordPress/gutenberg/pull/28216/files#diff-320e7dacc9e9a83ac8128d299319f82cf6dfa2609e0ebbc043d13de33ea92773R127)) puts them all on the same `assets/` folder.

Technically, I don't think that's a big deal. However, I can see how we lost a bit of context on those files.

If you think that we could benefit from having them in a dedicated `assets/` subfolder (and maybe can point me to a resource on how to make it happen 😅) I'll open a PR for it in Gutenberg and update this one accordingly.

Reply:
The context is nice, but I agree that it's not a big deal. 👍